tools/downloader/converter.py

```diff
@@ -107,8 +107,13 @@ def main():
         try:
             mo_path = Path(os.environ['INTEL_OPENVINO_DIR']) / 'deployment_tools/model_optimizer/mo.py'
         except KeyError:
-            sys.exit('Unable to locate Model Optimizer. '
-                + 'Use --mo or run setupvars.sh/setupvars.bat from the OpenVINO toolkit.')
+            # For OpenVINO from PyPI
+            try:
+                import mo
```

Comment:
This might not work, because MO will not necessarily be run with the same Python that Model Converter is run with. Perhaps you could do something like `subprocess.run([args.python, '-c', 'import mo; print(mo.__file__)'])` instead and capture the output.

Reply:
Good point. Fixed